pkg/landscaper/controllers/installations/reconcile_delete_test.go

```diff
@@ -134,4 +149,35 @@ var _ = Describe("Delete", func() {
 		Expect(instC.DeletionTimestamp.IsZero()).To(BeFalse())
 	})
 
+	It("should propagate the force deletion annotation to a execution in deletion state", func() {
+		ctx := context.Background()
+
+		var err error
+		state, err = testenv.InitResources(ctx, "./testdata/state/test3")
+		Expect(err).ToNot(HaveOccurred())
+
+		inst := &lsv1alpha1.Installation{}
+		inst.Name = "root"
+		inst.Namespace = state.Namespace
+		testutils.ExpectNoError(testenv.Client.Delete(ctx, inst))
+		_ = testutils.ShouldNotReconcile(ctx, ctrl, testutils.RequestFromObject(inst)) // returns a still waiting error
+
+		exec := &lsv1alpha1.Execution{}
+		exec.Name = "root"
+		exec.Namespace = state.Namespace
+		testutils.ExpectNoError(testenv.Client.Get(ctx, kutil.ObjectKeyFromObject(exec), exec))
+		Expect(exec.DeletionTimestamp).ToNot(BeNil())
+
+		testutils.ExpectNoError(testenv.Client.Get(ctx, kutil.ObjectKeyFromObject(inst), inst))
+		metav1.SetMetaDataAnnotation(&exec.ObjectMeta, lsv1alpha1.OperationAnnotation, string(lsv1alpha1.ForceReconcileOperation))
```

Comment:
I'm a little bit confused. Why setting the annotation at the execution and not the installation if you want to test its propagation?

Reply:
thanks good catch.
Obviously that test succeeds.

When I fixed that test it then fails. So I will also fix the Landscaper regarding that issue